dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.39/V2_39_16__Add_referral_parameter_to_program_stage.sql

```diff
@@ -0,0 +1,10 @@
+-- https://jira.dhis2.org/browse/DHIS2-13301
+-- Adding referral parameter to ProgramStage
+
+ALTER TABLE programstage ADD COLUMN IF NOT EXISTS referral BOOLEAN;
```

Comment:
I think Enrico recently added a migration like this. Does it look the same @enricocolasante ?